### PR TITLE
Fix 404 error on contest page refresh, created Alert component

### PR DESCRIPTION
**Pull request by MTsatsarov**
Closes https://github.com/SoftUni-Internal/exam-systems-issues/issues/832

### CODE REVIEW

**Review by shsimeonova — CHANGES_REQUESTED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/alert/Alert.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    return (
+        <Snackbar
+          open={open}
+          onClose={handleClose}
+          autoHideDuration={autoHideDuration}
+          anchorOrigin={{ vertical, horizontal }}
+          onClick={handleClose}
+        >
+            <MuiAlert
+              sx={{ backgroundColor: `${(severity === AlertSeverity.Success)}` && 'rgb(245, 78, 84)' }}
```
Comment (shsimeonova) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/alert/Alert.tsx:56]: this is odd

**Review by shsimeonova — CHANGES_REQUESTED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/alert/Alert.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
```
[the remark this answers is not in the document]
Comment (MTsatsarov) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/alert/Alert.tsx:1]: Yes

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/alert/Alert.tsx (hunk 1 above)
Comment (MTsatsarov) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/alert/Alert.tsx]: Refactored

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/alert/Alert.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    Center = 'center',
+}
+const Alert = (props:IAlertProps) => {
+    const { message, severity, variant, onClose, autoHideDuration, vertical, horizontal } = props;
+    const [ open, setOpen ] = useState<boolean>(true);
+
+    const alertClass = `${(severity === AlertSeverity.Success)}`
+        ? 'success-alert'
+        : '';
+    console.log(severity);
```
Comment (shsimeonova) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/alert/Alert.tsx:45]: Remove console log

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/alert/Alert.module.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.success-alert{
```
Comment (shsimeonova) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/alert/Alert.module.scss:1]: Space before opening bracket

**Review by shsimeonova — CHANGES_REQUESTED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/alert/Alert.module.scss (hunk 4 above)
Comment (MTsatsarov) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/alert/Alert.module.scss]: Done

**Review by shsimeonova — DISMISSED** [review]

**Review by shsimeonova — APPROVED** [review]

**Pull request merged**